### ISSUE TITLE
Please clear up the backend issue as soon as possible.

### ISSUE THREAD

**Issue by oskarsinger** [vispy/vispy#1116]
I did a quick Google search for 3d viz in bokeh, and it became clear that this is not currently possible. Somebody somewhere on the Internet suggested vispy for this task, so I checked out your page. 

Everything was going fine until I found that I needed one of these 'backend' things. I am one of the people in the 'scientists who don't understand OpenGL' category, so I didn't even know what this meant, much less which to choose out of the several you listed. This step alone is going to scare off an enormous set of potential users in the data science community.

Long story short, I now have a bunch of cruft in one of my miniconda environments, and I spent three hours trying to get various backends with backwater, buggy installation processes to work on my laptop.

I could have just given up and moved on, but think your project looks super cool, and I could not wait to use it as soon as I saw the galleries, so I am submitting this issue. As one of those people who needs something that 'just works', I am begging you all for the sake of the success of your project to fix this hitch in your installation process.

**Comment by rossant** [comment]
Admitedly the installation instructions could be clearer, especially when using conda (`conda install vispy`). For the backend it should be as simple as `conda install pyqt4`...

**Comment by oskarsinger** [comment]
I could not find conda-related instructions or any mention of a conda package anywhere on the github or .org sites. A quick Google search for 'conda install vispy' brings up no exact matches. 

It might help if you featured the fact that you have a conda package prominently somewhere on the github or .org website. 

Similar issues for PyQt4. All I can find is blog posts complaining how much of a pain it is to install SIP and PyQt4. They give instructions that don't work for me.

**Comment by rossant** [comment]
yes, see https://github.com/vispy/vispy/issues/1120

**Comment by oskarsinger** [comment]
Great, thank you.

**Comment by joseortiz3** [comment]
I think this is still an issue. I'm a new user and I can barely muster the patience to figure out how to get a backend working.
